development.ini

```diff
@@ -29,6 +29,9 @@ local_settings_path = %(here)s/local.ini
 
 node_modules = %(here)s/node_modules
 
+# path where pdf reports are archived
+pdf_archive_path = %(here)s/.build/reports
```

Comment:
We could use same data_path for pdf reports and hazardset layers.
But data_path is not set by default in pyramid config for now.